python/cudf/cudf/core/dataframe.py

```diff
@@ -712,6 +715,11 @@ def __init__(
 
                     self._data = new_df._data
                     self._index = new_df._index
+                    self._data._level_names = (
```

Comment:
Do we need some kind of helper to normalize this? I see this repeated a lot of times.

Reply:
Done 👍

Reply:
While the helper does reduce the amount of code somewhat, I think it reduces the overall comprehensibility sadly :(  I also find it a bit out-of-place for this helper to live in `column_accessor.py`.

I think either we should:

- Rethink the helper's name and API
- Go back to not having a helper (more explicit but also more code)

@bdice would you concur?

Reply:
I was on the fence about whether a helper was appropriate here. I feel pretty neutral about the implementation in 88ccc48 -- I would accept the PR with or without the helper, since the DRY benefit and benefit of explicit logic are about even to me. @shwina If you think we should revert that change, I'm fine with that. I don't have any clear improvements to propose for names or APIs of the helper.